extensions/packageManager/src/components/FeedModal.tsx

```diff
@@ -187,7 +187,7 @@ export const FeedModal: React.FC<WorkingModalProps> = (props) => {
           <Toggle
             ariaLabel={formatMessage('Include prerelease versions')}
             checked={item ? item.defaultQuery?.prerelease : false}
-            disabled={!selectedItem || selectedItem.readonly}
+            disabled={!selectedItem || item.key !== selectedItem.key || selectedItem.readonly}
```

Comment:
Can the first check be removed altogether? And should the `readonly` check be on the item, not the selected item?
```suggestion
            disabled={item.key !== selectedItem.key || item.readonly}
```

Reply:
I applied this change here and in a few other places.